### ISSUE TITLE
Essentia Dynamo - Not Accepting Essentia

### ISSUE THREAD

**Issue by HandGrenade** [theflogat/Technomancy#136]
I've been using the Essentia Dynamo (To Produce RF obviously), and there seems to be a bug with it accepting essentia. When I pipe essentia in to the Dynamo (With the 'Vanilla' Thaumcraft Essentia Pipes), it accepts fine when piped into the side of the Dynamo, but if I pipe from the bottom, it draws out 1 essentia (From 'vanilla' thaumcraft jars), but it simply disappears and doesn't show up in the dynamo or pipes.

Basically, It doesn't accept when piped from the bottom of the Dynamo.

Thanks :)

**Comment by Mordenkainen** [comment]
Since the dynamo can be rotated, can you clarify what you mean by "bottom"?

**Comment by HandGrenade** [comment]
By bottom, I would mean the only side which looks like an ordinary block (The side which is completely flat).

**Comment by Mordenkainen** [comment]
Does it just keep drawing essentia, or does it stop after drawing 1?

**Comment by HandGrenade** [comment]
It simply stops drawing at 1, and that single essentia doesn't even go into the dynamo, it just disappears from the pipes.

**Comment by Mordenkainen** [comment]
Does it behave the same if you rotate the dynamo?

**Comment by Mordenkainen** [comment]
I have just tested this and am unable to recreate.

**Comment by HandGrenade** [comment]
I've just tested again, in singleplayer, and am easily able to recreate with this setup (http://puu.sh/jhjC0/66619f297d.jpg) As you can see, I simply put in 16 aqua essentia, and it has withdrawn 1, but left the rest in there. I'm not sure of any current way to rotate the dynamo, so I cannot answer your first question.

**Comment by Mordenkainen** [comment]
The dynamo can be rotated with any wrench. It will automatically rotate to face any rf conductor or storage device adjacent to it when right clicked with just about any type of wrench.

What versions of Thaumcraft and Technomancy are you running?

**Comment by HandGrenade** [comment]
I've rotated it and it seems to now withdraw from the 'Full side', but it still will not withdraw if you pipe it from directly underneath, no matter what way it is rotated.

Thaumcraft - 4.2.3.5
Technomancy - 0.11.1

**Comment by Mordenkainen** [comment]
If you click the pipe with an essentia resonator what does it say?

**Comment by Mordenkainen** [comment]
Ah, ok. I see that you are using 0.11.1, not the 0.11.1B Beta.

In the 0.11.1B version changes have been made to how the Essentia Dynamo draws in Essentia. If possible, can you recheck your issue using 0.11.1B? I believe this issue is already fixed, and this is why I could not recreate it.

**Comment by Mordenkainen** [comment]
Closing since the user has not responded and I believe that is fixed in the latest beta.
